### PR TITLE
[CAM-13224]: use pylint and futurize to make code Python3-ready

### PR DESCRIPTION
**Pull request by grrtrr**
This adds changes to support Python3 in the APIv2 code.

### CODE REVIEW

File: src/clc/APIv2/server.py [hunk 1]
```diff
@@ -203,7 +204,7 @@ def Refresh(self):
 			self.data['changeInfo']['modifiedDate'] = clc.v2.time_utils.ZuluTSToSeconds(self.data['changeInfo']['modifiedDate'])
 
 			# API call switches between GB and MB.  Change to all references are in GB and we drop the units
-			self.data['details']['memoryGB'] = int(math.floor(self.data['details']['memoryMB']/1024))
+			self.data['details']['memoryGB'] = int(math.floor(self.data['details']['memoryMB'] // 1024))
```
Comment (EfrenRey) [inline src/clc/APIv2/server.py]: Minor comment. As you use floor division operator (`//` ), I think you can remove `math.floor`.
```python
self.data['details']['memoryGB'] = int(self.data['details']['memoryMB'] // 1024)
```

Comment (grrtrr) [reply src/clc/APIv2/server.py]: > @grrtrr Please also update https://github.com/CenturyLinkCloud/clc-python-sdk/blob/master/README.md where it currently says no support for Python-3 and add some proper instructions for the support of both python-2.7 and python-3.

I have removed the comment saying that Python3 is not supported. The PR [has the necessary modifications for listing on PyPi](https://github.com/CenturyLinkCloud/clc-python-sdk/pull/49/files/08a4a71b59d3dcbe3541399d9dbca6a2e6ea65ae#diff-ce2555bbe69b610cc512e39f38430319R13-R39), when they submit to [clc-sdk](https://pypi.org/project/clc-sdk/), this should mark
the package as Python3-enabled.

**Comment by grrtrr** [comment]
> @grrtrr Please also update https://github.com/CenturyLinkCloud/clc-python-sdk/blob/master/README.md where it currently says no support for Python-3 and add some proper instructions for the support of both python-2.7 and python-3.

I have removed the comment saying that Python3 is not supported. The PR [has the necessary modifications for listing on PyPi](https://github.com/CenturyLinkCloud/clc-python-sdk/pull/49/files/08a4a71b59d3dcbe3541399d9dbca6a2e6ea65ae#diff-ce2555bbe69b610cc512e39f38430319R13-R39), when they submit to [clc-sdk](https://pypi.org/project/clc-sdk/), this should mark the package as Python3-enabled.

**Comment by jinzishuai** [comment]
test results of the CAM Cloud Tests in CLC running in a python-3 testbed after removing the billing code (https://github.com/jinzishuai/elasticbox/commit/99e651d9cabeeb1a198dffea37d85f2c10810116)
http://10.55.220.31:8080/view/CAM/job/CAM-Platform-ALM/job/ALM-Appliance-CloudTest/28/testReport/
image

**Pull request merged**